server_modules/args.js

```diff
@@ -0,0 +1,7 @@
+var argv = require('minimist')(process.argv.slice(2));
+
+exports.port = argv.p || 1390;
+exports.basicAuthCreds = argv.u;
+exports.datadir = argv.d || 'data';
+exports.slaveMode = argv.s || false;
+exports.redisPort = argv.r || 13903;
```

Comment:
It's there from when I tried to use it for the session, but it didn't work very well...
I think that file based sessions are the best way here, but we need to see how we can get them to work. This will be in a different PR, sense it's its own thing (as I see it)